libraries/provision/ansible/playbooks/sgcollect-info.yml

```diff
@@ -14,9 +14,6 @@
   - include: tasks/sg-sgcollect-info-windows.yml
     when: ansible_os_family == "Windows"
 
-  - include: tasks/sg-pullRedacted-zip-windows.yml
```

Comment:
why we removed this task?

Reply:
pullling redacted zip file is part of  sg-sgcollect-info-windows.yml, so it is removed.

Reply:
pullling redacted zip file is part of  sg-sgcollect-info-windows.yml, so it is removed.